go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/DelineaXPM/tss-sdk-go/v2
 
 go 1.13
+
+require (
+	github.com/patrickmn/go-cache v2.1.0+incompatible // indirect
```

Comment:
@pacificcode are we using these dependencies? it is showing me unused references.